### PR TITLE
Local adjustments - denoise improvments

### PR DESCRIPTION
**Pull request by Desmis**
This PR is an attempt  to address some of the usability issues raised by a user on Pixls.us.

The user interface has been re-organised and a panel has been added to provide an indication of the residual noise levels when carrying out denoise adjustments.

Jacques

### CODE REVIEW

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -96,8 +96,6 @@ jobs:
 
       - name: Bundle dependencies
         run: |
-          echo "Listing shared library dependencies."
-          ldd "./build/${{matrix.build_type}}/rawtherapee.exe"
```
Comment (Lawrence37) [inline .github/workflows/windows.yml]: Restore this too before merging

Comment (Desmis) [reply .github/workflows/windows.yml]: I think you have made this change.

Thank you

Jacques

**Comment by Lawrence37** [comment]
@Desmis In this comment https://discuss.pixls.us/t/chrominance-noise-reduction-causes-size-dependent-desaturation/35582/8 you indicated the automatic chrominance noise reduction is reduced when the local adjustments denoise is enabled. I think there should be no change otherwise it can be confusing and we would have to ensure reproducibility of older edits. Anyone who uses both noise reduction tools together should already be aware that their effects stack and can make adjustments while taking that into consideration.

**Comment by Desmis** [comment]
@Lawrence37 

OK, i will change

Jacques

**Comment by Desmis** [comment]
If nobody objects, I will merge this PR in the next days

Jacques

**Pull request merged**